packages/slate-editor/src/components/EditorBlock/Menu.tsx

```diff
@@ -6,78 +7,93 @@ import { Popper } from 'react-popper';
 
 import { Toolbox } from '#components';
 
+import type { PopperOptionsContextType } from '#modules/popper-options-context';
+
 import styles from './Menu.module.scss';
 
 interface Props {
     children: ReactNode;
     className?: string;
     onClick?: (event: MouseEvent) => void;
+    popperOptions: PopperOptionsContextType;
     reference: HTMLElement;
 }
 
-const VIRTUAL_REFERENCE_SIZE = 42;
+const TETHER_OFFSET_AMOUNT = 4;
+const TETHER_OFFSET_OUTLINE_SIZE = 6;
```

Comment:
I've made the menu align to the outline, currently it was a few pixels off (somewhere between the element and its outline).

I quite like it but wasn't sure if we want it or if it should be aligned to the element rather than the outline, so let me know!